data/morphology/grab_unimorph_data.py

```diff
@@ -1,52 +1,69 @@
 #!/usr/bin/env python
 """Downloads UniMorph morphological paradigms data."""
-
 import json
 import logging
 import os
-import requests
 import time
-
 from typing import Dict
 
 
+""" import requests in separate code block because of third party status """
+import requests
+
+
+start_time = time.time()
+
+
 UNIMORPH_DICT_PATH = "unimorph_languages.json"
 
 
-def download(data_to_grab: Dict[str, str]) -> Dict[str, str]:
-    to_retry = {}
-    os.mkdir("tsv")
-    for language, url in data_to_grab.items():
-        with requests.get(url, stream=True) as response:
-            logging.info("Downloading: %s", language)
+def download():
+    os.makedirs("unimorph_for_split", exist_ok=True)
+    with open(UNIMORPH_DICT_PATH) as jfile:
+        data_to_grab = json.load(jfile)
+    urls = list(data_to_grab.values())
+    lg_codes = list(data_to_grab.keys())
+    for i in range(len(urls)):
+        with requests.get(urls[i], stream=True) as response:
+            target_path = urls[i].split("/")[-1]
+            logging.info("Downloading: %s", target_path)
             if response.status_code == 200:
-                with open(f"tsv/{language}.tsv", "wb") as f:
-                    f.write(response.content)
+                with open(f"unimorph_for_split/{target_path}.tsv", "w+") as sink:
+                    print(response.text,file=sink)
+            elif target_path == "glg":
```

Comment:
I have no clue what lines 34-44 are supposed to correspond to. Is this testing code?

Reply:
here I made 2 conditions because the language codes in wikipron and unimorph don't match and I wanted to maintain a path to the files that was the same as the unimorph lg code even though the lg codes in the keys() of the dict are wikipron conventions specifically for galician and georgian. Should I keep this or change the language codes in the tsv file paths to match wikipron?